tests/unittests/net/test_dhcp.py

```diff
@@ -512,6 +521,100 @@ def test_dhcp_discovery(self, m_wait, m_subp, m_kill, m_getppid, m_remove):
             ]
         )
         m_kill.assert_has_calls([mock.call(my_pid, signal.SIGKILL)])
+        mocked_is_ib_interface.assert_called_once_with("eth9")
+
+    @mock.patch("cloudinit.temp_utils.get_tmp_ancestor")
```

Comment:
Style nitpick: Mocks are usually easier to read when the return value is defined with the mock declaration like this:
```
    @mock.patch("cloudinit.temp_utils.get_tmp_ancestor", return_value="/tmp")
```

For this new one could we make that change to these mocks please?